<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackOffice - Patatestore</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_onglets">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="produits">
       <attribute name="title">
        <string>Produits</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_nom">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBox_variete"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_nomProduit"/>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QTableWidget" name="tableWidget_produits">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Numéro</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantité</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Informations</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Image</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Variété</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_quantitee"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_image">
          <property name="text">
           <string>Image :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="lineEdit_informations"/>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="pushButton_ajouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="lineEdit_image"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_quantite">
          <property name="text">
           <string>Quantité :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_variete">
          <property name="text">
           <string>Variété :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_informations">
          <property name="text">
           <string>Informations :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_supprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButton_modifier">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="varietes">
       <attribute name="title">
        <string>Variétés</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="rayons">
       <attribute name="title">
        <string>Rayons</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="producteurs">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <attribute name="title">
        <string>Producteurs</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
